content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -53,7 +53,7 @@ A [reference implementation](https://github.com/envoyproxy/ratelimit) of the API
       name: ratelimit-config
     data:
       config.yaml: |
-        domain: ratelimit
+        domain: product # domain name is overridden at route filter level
```

Comment:
Same here, why is this being overridden? The numbered list item does not provide any context here.